lib/fog/parsers/terremark/get_vdc.rb

```diff
@@ -9,74 +9,74 @@ def reset
           @in_instantiated_vms_quota = false
           @in_deployed_vms_quota = false
           @response = {
-              'links' => [],
-              'AvailableNetworks' => [],
-              'ComputeCapacity'   => {
-                  'Cpu' => {},
-                  'DeployedVmsQuota' => {},
-                  'InstantiatedVmsQuota' => {},
-                  'Memory' => {}
-              },
-              'StorageCapacity'  => {},
-              'ResourceEntities' => []
+            "links" => [],
+            "AvailableNetworks" => [],
+            "ComputeCapacity"   => {
+                "Cpu" => {},
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.